### ISSUE TITLE
Emoji statuses for linting

### ISSUE THREAD

**Issue by ewels** [nf-core/tools#2538]
The "fix linting" automation off a comment is great, but you are sort of blind about whether it's worked or not until a commit lands.

It'd be great to add to the action steps that add emoji reactions to the initiating comment, so that you have an idea if it's been correctly triggered, or failed.

* 👀 As soon as the action starts
* ❌ If the action fails (or a comment with the action log??)
* ✅ If the action completed successfully (including if no changes were made / committed).

**Comment by mashehu** [comment]
Partially already done in modules: https://github.com/nf-core/modules/blob/master/.github/workflows/fix-linting.yml

**Comment by ewels** [comment]
Nice! I had a feeling that was the case but couldn't find it. Should've checked modules. Ok then it's super easy - just copy this across to the pipeline template / tools / website repos then 😊
